### DISCUSSION TITLE
change the authorized ADE ID account

### DISCUSSION THREAD

**Discussion by maudonna** [Leseratte10/acsm-calibre-plugin#49] [General]
Hi, I setup the plugin by entering my usual adobe ID account credentials, then I read on and decided to apply your advise of using a dummy account but the authorize account can't be change, even after removing the Export account activation data file and the Export account encryption key file. I also tried removing the plugin and reinstalling it from a new download, to no avail. How could I reset the AdobeId account?

Thanks

**Comment by Leseratte10** [comment]
Same as in ADE - open up the plugin settings, then press CTRL+SHIFT+D and follow the prompts.

**Comment by sourcecodemage** [comment]
I'm on Calibre for Mac in the Customize plugin screen for DeACSM plugin. CTRL+SHIFT+D  doesn't appear to do anything. Here is what the customize plugin screen looks like. 

<img width="783" height="476" alt="deacsm_config_screen" src="https://github.com/user-attachments/assets/52592657-d9d8-437e-835c-ae2f80193466" />

**Reply by sourcecodemage** [reply]
ok. For Mac, it is cmd+shift+d to activate the deauthorization popup screen.
